src/upload_all_java_tools.sh

```diff
@@ -48,7 +48,7 @@ bazel_version=$(bazel info release | cut -d' ' -f2)
 
 # Passing the same commit_hash and timestamp to all targets to mark all the artifacts
 # uploaded on GCS with the same identifier.
-for java_version in 11; do
+for java_version in 11 15; do
```

Comment:
Also from here. This is for releasing it.

Reply:
Sorry, done